htdocs/admin/pdf.php

```diff
@@ -399,6 +401,21 @@
 }
 print '</td></tr>';
 
+// Show alias in thirdparty name
+print '<tr class="oddeven"><td>'.$langs->trans("PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME").'</td><td>';
+if ($conf->use_javascript_ajax) {
+	$arrval = array('0' => $langs->trans("No"), '1' => $langs->trans("THIRDPARTY_ALIAS"), '2' => $langs->trans("ALIAS_THIRDPARTY"));
+	print $form->selectarray("PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME", $arrval, getDolGlobalInt('PDF_INCLUDE_ALIAS_IN_THIRDPARTY_NAME'));
+}
+
+print '<tr class="oddeven"><td>'.$langs->trans("MAIN_USE_COMPANY_NAME_OF_CONTACT").'</td><td>';
```

Comment:
The default behaviour is already tha same as MAIN_USE_COMPANY_NAME_OF_CONTACT = 1
So to avoid non usefull options inflation, we don't need to have this option available into setup, the default behaviour is already to have it enabled.

Reply:
ok it's done